tests/queries/0_stateless/00080_show_tables_and_system_tables.sql

```diff
@@ -1,25 +1,24 @@
--- Tags: no-parallel
 
-DROP DATABASE IF EXISTS test_show_tables;
+DROP DATABASE IF EXISTS {CLICKHOUSE_DATABASE:Identifier};
 
-CREATE DATABASE test_show_tables;
+CREATE DATABASE {CLICKHOUSE_DATABASE:Identifier};
 
-CREATE TABLE test_show_tables.A (A UInt8) ENGINE = TinyLog;
-CREATE TABLE test_show_tables.B (A UInt8) ENGINE = TinyLog;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.A (A UInt8) ENGINE = TinyLog;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.B (A UInt8) ENGINE = TinyLog;
 
-SHOW TABLES from test_show_tables;
+SHOW TABLES from {CLICKHOUSE_DATABASE:Identifier};
 SHOW TABLES in system where engine like '%System%' and name in ('numbers', 'one');
 
-SELECT name, toUInt32(metadata_modification_time) > 0, engine_full, create_table_query FROM system.tables WHERE database = 'test_show_tables' ORDER BY name FORMAT TSVRaw;
+SELECT name, toUInt32(metadata_modification_time) > 0, engine_full, create_table_query FROM system.tables WHERE database = currentDatabase() ORDER BY name FORMAT TSVRaw;
 
 CREATE TEMPORARY TABLE test_temporary_table (id UInt64);
 SELECT name FROM system.tables WHERE is_temporary = 1 AND name = 'test_temporary_table';
 
-CREATE TABLE test_show_tables.test_log(id UInt64) ENGINE = Log;
-CREATE MATERIALIZED VIEW test_show_tables.test_materialized ENGINE = Log AS SELECT * FROM test_show_tables.test_log;
+CREATE TABLE {CLICKHOUSE_DATABASE:Identifier}.test_log(id UInt64) ENGINE = Log;
+CREATE MATERIALIZED VIEW {CLICKHOUSE_DATABASE:Identifier}.test_materialized ENGINE = Log AS SELECT * FROM {CLICKHOUSE_DATABASE:Identifier}.test_log;
 SELECT dependencies_database, dependencies_table FROM system.tables WHERE name = 'test_log';
```

Comment:
I think the `dependencies_database` makes this query unusable in parallel tests, but it can be rephrased to work properly:

```
SELECT dependencies_table FROM system.tables WHERE name = 'test_log' AND dependencies_database = {CLICKHOUSE_DATABASE:String};
```

This way we can ensure that `dependencies_database` has the correct value but it can be determined in runtime.